<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Linna An homepage</title>
    <link rel="stylesheet" href="styles.css">
</head>


<body>
    <div class="container">
        <div class="topnav">
            <button id="toggle-mode">Light</button>
            <a href="index.html#contact">contact</a>
            <a href="index.html#news">news</a>
            <a href="index.html#Publication">publication</a>
            <a href="index.html#profile">about</a>
            
            
        </div>
        <div class="header">
            <h1>Personal Page for Linna An, Ph.D</h1>
            <h2>- Integrate AI and experiments -</h2>
        </div>
        <div class="center">
            <div class="contact">
                
                <h2><div id="contact"><a name="contact"></a>Contact Me</a></div></h2>
                <ul>
                    <li>Email: linnaan[a]uw.edu</li>
                    <li>
                      <strong><a href="LA_CV.pdf">Curriculum Vitae [click for pdf version]</a></strong>
                    </li>
                    <li>
                      <a href="https://github.com/LAnAlchemist" target="_blank">
                        <ion-icon name="logo-github" size="large"></ion-icon>
                      </a>
                      <a href="https://www.linkedin.com/in/linna-an-2539b739/" target="_blank">
                        <ion-icon name="logo-linkedin" size="large"></ion-icon>
                      </a>
                      <a href="https://twitter.com/alchemist_an" target="_blank">
                        <ion-icon name="logo-twitter" size="large"></ion-icon>
                      </a>
                      <a href="https://scholar.google.com/citations?user=U-_jk7oAAAAJ&hl=en" target="_blank">
                        <ion-icon name="logo-google" size="large"></ion-icon></ion-icon>
                      </a>
                    </li>
                    
                </ul>
            </div>
        </div>
        <div class="profile">
            
            <div class="profile-content">
                <h2><div id="profile"><a name="profile">The research career of Linna An</a></div></h2>
                <img src="linnaan.png" alt="LA photo" class="profile-image">
                <p>I did my undergrad at the <strong><em></em><a href="https://en.ustc.edu.cn/" target="_blank">University of Science and Technology of China (USTC)</a></em></strong> with Prof. Gaolin Liang (now moved to SEU) on molecular imaging.</p>
                <p>Then, I was trained as a biochemist during Ph.D. at the <strong><em><a href="https://www.vanderdonk.scs.illinois.edu/" target="_blank">Wilfred A. van der Donk lab</a></em></strong> at the <a href="https://chemistry.illinois.edu/" target="_blank"> University of Illinois Urbana-Champaign (UIUC)</a>. I worked on using bioinformatics to find new biosynthetic gene clusters, and characterize their synthesis product in lab. This is a new way to find new drugs from nature, including antibiotics. I successfully heterologously identified some new natrual products. Particularly, I also unveiled a unique substrate-assisted bond formation mechanism in a previously unknown family of enzymes <a href="https://www.nature.com/articles/s41589-018-0122-4" target="_blank">(An, Nat. Chem. Biol. 2018)</a>.</p>
                <p>Right before COVID, I joined the <strong><em><a href="https://www.bakerlab.org/" target="_blank">David Baker lab</a></em></strong> at University of Washtingon (UW) as a protein designer, developing deep learning-based method to design functional proteins. </p>
                <p>
                    I focused on <strong>small molecule-protein interaction problems</strong> because it is the unavoidable mountain to climb for solving any ligand-related engineering problem. It was difficult to deal with back in 2020, and some challenges remain till today: the large sampling space, the high accuracy requirements, and small datasets. I came up with a probably obvious idea: if we can generate proteins with high shape complementary interfaces to ligands, we should be able to bind them. Even though it was an seemingly obvious and easy idea, it took us 3.5 years to actually make it happen in reality, during which almost all steps are optimized or equipped with new tools.
                </p>
                <p>
                    I first teamed up with Dr. Derrick R . Hicks and Dr. Dmitri Zorine (now developer at <a href="https://x.com/ZipBio_Inc" target="_blank">ZipBio, Inc</a>), and we developed an AF2-based hallucination method, which can efficiently sample diverse pocket-containing proteins. These proteins are high quality, and ready to bind any shape of ligands <strong><a href="https://www.nature.com/articles/s41594-023-01112-6" target="_blank">(An, Hicks, Zorine, et al, Nat. Struct. Mol. Biol, 2023)</a></strong>.
                </p>
                <p>
                    With high-quality proteins, I then worked on the actual ligand binding problems. I first computationally identified the most shape complementary protein docking to the ligand, then in the laboratory, using high-throughput methods to identify the ones who are actually suitable to the ligand. From the privileged protein-ligand pair, with computational sampling of local structures and docking conformations, I  was able to quickly improve the binding affinity to nanomolar range. This pipeline stably generated binders to diverse ligands, including polar and flexible ligands, such as methotrexate, for the first time!  We easily converted the de novo binders into sensors, and showed their usage in ligand-specific nanopore and chemical induced dimerization. <strong><a href="https://www.science.org/doi/10.1126/science.adn3780?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed" target="_blank">(An, Science, 2024)</a></strong>.
                </p>
                <p>
                    More recently, with my colleagues, we successfully implemented the whole idea to Ca-Diffusion (<em>unpublished</em>), and we demonstrated it carried over its success in more ligands stably, and stayed specific in cells.
                </p>
                <p>
                    Another direction I am pushing forward is, using the small-molecule protein manipulation pipeline I developed to engineer enzymes. Stay tuned for all interesting results! Ultimately, I want to build my team and/or form a team with others, in both acamedia and industry, tackle questions of sensing and enzyme domestication!
                </p>
                <p>
                    <u><a href="index.html#contact">Currently, I am in the job market. Do let me know if you are interested!</a></u>
                </p>
            </div>
        </div>
        
        <div class="skills">
            <h2><div id="news"><a name="news">News</a></div></h2>
            <blockquote class="twitter-tweet"><p lang="en" dir="ltr">Check out how we use ML-assisted method to design small molecule binders (binder to polar and flexible ligands as well!) and turn binding to sensing via CID and nanopore. <a href="https://twitter.com/UWproteindesign?ref_src=twsrc%5Etfw">@UWproteindesign</a> Art credit to <a href="https://twitter.com/ichaydon?ref_src=twsrc%5Etfw">@ichaydon</a> | Science <a href="https://t.co/N4jUZFVvfQ">https://t.co/N4jUZFVvfQ</a> <a href="https://t.co/DkWKceTEfR">pic.twitter.com/DkWKceTEfR</a></p>&mdash; Linna An (@alchemist_an) <a href="https://twitter.com/alchemist_an/status/1814022913064054808?ref_src=twsrc%5Etfw">July 18, 2024</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
            <blockquote class="twitter-tweet"><p lang="en" dir="ltr">Thanks to all the organizers! <a href="https://t.co/hKTOw7NNJD">https://t.co/hKTOw7NNJD</a></p>&mdash; Linna An (@alchemist_an) <a href="https://twitter.com/alchemist_an/status/1834764019619483689?ref_src=twsrc%5Etfw">September 14, 2024</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
        </div>
        
        <!-- 240920 add -->
        <div class="skills">
            <h2><div id="Publication"><a name="Publication">Selected Publication</a></div></h2>
            <div class="publication-list">
                <div class="publication">
                    <input type="checkbox" id="pub20">
                    <label for="pub20">20. J. Dauparas, G. Lee, R. Pecoraro, <strong>L. An</strong>, I. Anishchenko, C. Glasscock, D. Baker*, Atomic context-conditioned protein sequence design using LigandMPNN, 2024, Nature Methods, Accepted.</label>
                    <div class="content">
                        <p>Ligand-aware protein sequence design neural network.</p>
                        <ul>
                            <li><a href="https://www.google.com/url?q=https://doi.org/10.1101/2023.12.22.573103&sa=D&source=docs&ust=1726877280394471&usg=AOvVaw210cfeqYuus7oHUeuUChEW" target="_blank">Preprint</a></li>
                        </ul>
                    </div>
                </div>
    
                <div class="publication">
                    <input type="checkbox" id="pub19">
                    <label for="pub19">19. <strong>L. An*</strong>, M. Seid, L. Tran, S. Majumder, I. Goreshnik, G. Lee, J. Dauparas, I. Anishchenko, B. Coventry, A. Bera, A. Kang, P. M. Levine, V. Alvarez, A. Pillai, C. Norn, D. Feldman, D. Zorine, D. R. Hicks, X. Li, M. G. Sanchez, D. K. Vafeados, P. J. Salveson, A. A. Vorobieva, and D. Baker*. Binding and sensing diverse small molecules using shape complementary pseudocycles, 2024, Science, 385, 276.</label>
                    <div class="content">
                        <p>Ligand-aware protein sequence design neural network.</p>
                        <ul>
                            <li><a href="https://www.science.org/doi/10.1126/science.adn3780" target="_blank">on Science</a></li>
                            <li><a href="https://www.bakerlab.org/2024/07/17/designing-synthetic-smell-receptors/" target="_blank">Report from Baker lab</a></li>
                            <li><a href="https://www.chemistryworld.com/news/nanopore-proteins-designed-from-scratch-and-turned-into-biosensors/4019873.article" target="_blank">News from Chemistry World</a></li>
                        </ul>
                    </div>
                </div>
                <div class="publication">
                    <input type="checkbox" id="pub18">
                    <label for="pub18">18. <strong>L. An#*</strong>, D. Hicks#, D. Zorine#, J. Dauparas, B. Wicky, L. Milles, A. Courbet, A. Bera, H. Nguyen, A. Kang, L. Carter, D. Baker*. Hallucination of closed repeat proteins containing central pockets, Nat. Struct. Mol. Biol., 2023, 30, 1755.</label>
                    <div class="content">
                        <p>Computational sampling of pocket containing proteins.</p>
                        <ul>
                            <li><a href="https://www.nature.com/articles/s41594-023-01112-6#citeas" target="_blank">on Nature-Springer</a></li>
                        </ul>
                    </div>
                </div>
                <div class="publication">
                    <input type="checkbox" id="pub17">
                    <label for="pub17">17. <strong>L. An#</strong>, D. Cogan#, C. Navo, G. Jiménez-Osés, S. K. Nair, and W. A. van der Donk*, Substrate-assisted enzymatic formation of lysinoalanine in duramycin, Nat. Chem. Biol. 2018, 10, 928.</label>
                    <div class="content">
                        <p>Mechanism of action study of a unique substrate-assisted enzyme.</p>
                        <ul>
                            <li><a href="https://www.nature.com/articles/s41589-018-0122-4.pdf" target="_blank">on Nature-Springer</a></li>
                        </ul>
                    </div>
                </div>
                <div class="publication">
                    <input type="checkbox" id="pub16">
                    <label for="pub16">16. <strong>L. An#*</strong> and G. R. Lee#*, De novo protein design using the blueprint builder in Rosetta, Current Protocols in Protein Science, 2020, 102,1, e116.</label>
                    <div class="content">
                        <p>Ligand-aware protein sequence design neural network.</p>
                        <ul>
                            <li><a href="https://currentprotocols.onlinelibrary.wiley.com/doi/10.1002/cpps.116" target="_blank">on Current Protocols</a></li>
                        </ul>
                    </div>
                </div>
            </div>
            <script src="scripts.js"></script>
        </div>

        <button onclick="topFunction()" id="myBtn" title="Go to top">Top</button>

        <div class="footer">
            <p>© 2024 Linna An. All rights reserved.</p>
        </div>
    </div>
<!--add to allow use of logo https://ionic.io/ionicons/usage-->
<script type="module" src="https://unpkg.com/ionicons@7.1.0/dist/ionicons/ionicons.esm.js"></script>
<script nomodule src="https://unpkg.com/ionicons@7.1.0/dist/ionicons/ionicons.js"></script>
<script> // button for go to top
// Get the button
let mybutton = document.getElementById("myBtn");

// When the user scrolls down 20px from the top of the document, show the button
window.onscroll = function() {scrollFunction()};

function scrollFunction() {
    if (document.body.scrollTop > 20 || document.documentElement.scrollTop > 20) {
    mybutton.style.display = "block";
    } else {
    mybutton.style.display = "none";
    }
}

// When the user clicks on the button, scroll to the top of the document
function topFunction() {
    document.body.scrollTop = 0;
    document.documentElement.scrollTop = 0;
}
</script>
<script src="day_night_theme.js"></script>
</body>
</html>
